engine/engine.go

```diff
@@ -897,12 +897,36 @@ func (bot *Engine) SetupExchanges() error {
 		bot.dryRunParamInteraction("exchangehttpdebugging")
 	}
 
+	var exchangesOverride []string
+	if bot.Settings.Exchanges != "" {
+		bot.dryRunParamInteraction("exchanges")
+		exchangesOverride = strings.Split(bot.Settings.Exchanges, ",")
+		for x := range exchangesOverride {
+			if !common.StringDataCompareInsensitive(exchange.Exchanges, exchangesOverride[x]) {
+				return fmt.Errorf("exchange %s not found", exchangesOverride[x])
+			}
+		}
+	}
+
```

Comment:
Suggestion:
```diff
diff --git a/engine/engine.go b/engine/engine.go
index a0423819f..75339e1a7 100644
--- a/engine/engine.go
+++ b/engine/engine.go
@@ -908,6 +908,10 @@ func (bot *Engine) SetupExchanges() error {
                }
        }

+       if bot.Settings.EnableAllExchanges && len(exchangesOverride) > 0 {
+               return errors.New("cannot enable all exchanges and specific exchanges concurrently")
+       }
+
        var wg sync.WaitGroup
        for x := range configs {
                shouldLoad := false
```

Reply:
Yeah I did think of this but didn't spank the user if they set both options, it's probably best if we do though 😈 Also included another change which fixes suppressed warnings for multiple dry run inducing commands instead of just on the single occurrence